### ISSUE TITLE
Installer and setting locales to en_US.UTF-8

### ISSUE THREAD

**Issue by interduo** [mail-in-a-box/mailinabox#1762]
`setup/start.sh: line 23: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8): No such file or directory`

Is there really need to change locales?

**Comment by guyzmo** [comment]
yes, it needs to because most of the tools are assuming UTF-8 as default, and thus not supporting it would lead to errors and crashes. I'm not talking about mailinabox specifically, but generally speaking, the issue would rather be ubuntu not using UTF-8 locales per default ;-)
